docs_list.md

```diff
@@ -0,0 +1,143 @@
+# Docs list
+
+A working list of docs with links to the published documentation, the repo or repos, and the intended purpose of the technical content.
+
+## Cosmos Hub
+
+- [Cosmos Hub](https://hub.cosmos.network/main/hub-overview/overview.html)
+
+- <https://github.com/cosmos/gaia/tree/master/docs>
+
+- Using the Cosmos SDK application for the Cosmos Hub.
+
+  <details><summary>Top-level contents</summary>
+
+  - Cosmos Hub
+
+  - Delegators
+
+  - Gaia Tutorials
+
+  - Validators
+
+  </details>
+
+## Cosmos SDK Documentation
+
+- [Cosmos SDK Documentation](https://docs.cosmos.network/) _why 2 repos?_
+
+- <https://github.com/allinbits/cosmos-sdk/tree/master/docs>
+
+- <https://github.com/cosmos/cosmos-sdk/tree/master/docs> <!-- two repos, what is the difference? -->
+
+- Learn about open source Cosmos SDK framework.
+
+  <details><summary>Top-level contents</summary>
+
+    - Introduction
+
+    - Basics
+
+    - Core Concepts
+
+    - Building Modules
+
+    - IBC
+
+    - Running a Node, API and CLI
+
+    - Migrations
+
+    - USING THE SDK
+
+      - Modules
+
+        - List of Modules
+
+    </details>
+
+## Cosmos SDK Tutorials
+
+- [Cosmos SDK Tutorials](https://tutorials.cosmos.network/)
+
+- <https://github.com/cosmos/sdk-tutorials>
+
+- Tutorials for building modules for the Cosmos SDK. Links to videos.
+
+  <details><summary>Top-level contents</summary>
+
+    - Voter
+
+    - Blog
+
+    - Blog (legacy)
+
+    - PoFE
+
+    - PoFE - Launchpad to Stargate
+
+    - Scavenge
+
+    - Nameservice
+
+    - Cosmos Burner Chain
+
+  </details>
+
+## Starport
+
+- [Tendermint Starport](https://github.com/tendermint/starport/tree/develop/docs)
+
+- <https://github.com/tendermint/starport>
+
+- Tool to bootstrap blockchains. Developer-friendly, easiest way to build a blockchain.
+
+- Note: View the docs only in the repo or the [gitpod workspace](https://gitpod.io/#https://github.com/tendermint/starport/tree/master).
+
+  <details><summary>Top-level contents</summary>
+
+  - Introduction
+
+  - Install
+
+  - Quickstart
+
+  - Serving a blockchain
+
+  - Scaffolding
+
+  - Configuration
+
+  - Relayer
+
+  </details>
+
+## Tendermint Core
+
+- [Tendermint Core](https://docs.tendermint.com/master/)
+
+- <https://github.com/tendermint/tendermint/tree/master/docs>
+
+- Blockchain application platform, BFT state machine replication.
```

Comment:
```suggestion
PFBT Consensus Engine used by the Cosmos SDK and other blockchain applications.
```

Reply:
adding to batch new desc for Tendermint Core